lib/panoptes/client/version.rb

```diff
@@ -1,5 +1,5 @@
 module Panoptes
   class Client
-    VERSION = "0.3.8".freeze
+    VERSION = "0.3.9".freeze
```

Comment:
Seems like a minor version change vs patch as we are adding new functionality into the client.

Reply:
That seems totally reasonable, I just didn't want to get boastful :)